.travis.yml

```diff
@@ -80,6 +80,22 @@ matrix:
         - go get github.com/bazelbuild/buildtools/buildifier
         - ./ci/travis/bazel-format.sh
 
+    - os: linux
+      env: SANITIZER=1 CC=clang PYTHON=3.5 PYTHONWARNINGS=ignore
+
+      install:
+        - eval `python $TRAVIS_BUILD_DIR/ci/travis/determine_tests_to_run.py`
+        - if [ $RAY_CI_PYTHON_AFFECTED != "1" ]; then exit; fi
+
+        - ./ci/suppress_output ./ci/travis/install-bazel.sh
+        - ./ci/suppress_output ./ci/travis/install-dependencies.sh
+        - export PATH="$HOME/miniconda/bin:$PATH"
```

Comment:
It's best to prefer prefixed env vars unless there's a reason not to (like redundancy, etc.):

    - PATH="$HOME/miniconda/bin:$PATH" ./ci/suppress_output ./ci/travis/install-ray.sh  

That way the scope of each modification is explicit.

Reply:
ditto